app/src/main/java/nl/nfi/cellscanner/recorder/RecorderUtils.java

```diff
@@ -81,6 +81,12 @@ public static boolean exportMeteredAllowed(Context context) {
                 .getBoolean(PreferencesActivity.PREF_UPLOAD_ON_WIFI_ONLY, false);
     }
 
+    public static boolean exportScheduled(Context context) {
```

Comment:
Actually this method does not return whether an export has been scheduled or not, but whether the user wants this to be done.

Reply:
changed name and way to schedule periodic work

Reply:
-> Renamed method
-> Schedule periodic work now runs enqueueUniquePeriodicWork 
"This method allows you to enqueue a uniquely-named PeriodicWorkRequest, where only one PeriodicWorkRequest of a particular name can be active at a time. For example, you may only want one sync operation to be active. If there is one pending, you can choose to let it run or replace it with your new work."